src/components/TorrentDetail/Peers.vue

```diff
@@ -162,8 +162,13 @@ onBeforeRouteUpdate(() => !addPeersDialog.value)
 
       <template #[`item.country`]="{ item }">
         <div v-if="showCountryFlags" class="cursor-help" :title="item.country">
-          <img v-if="isWindows" :alt="codeToFlag(item.country_code!).char" :src="codeToFlag(item.country_code!).url" :title="item.country" style="max-width: 32px" />
-          <span v-else :title="item.country">{{ codeToFlag(item.country_code!).char }}</span>
+          <template v-if="item.country_code">
+            <img v-if="isWindows" :alt="codeToFlag(item.country_code!).char" :src="codeToFlag(item.country_code!).url" :title="item.country" style="max-width: 32px" />
+            <span v-else :title="item.country">{{ codeToFlag(item.country_code!).char }}</span>
+          </template>
+          <template v-else>
+            <span></span>
+          </template>
```

Comment:
No need for an extra template, just slap the condition on top of others. With this, we can remove the non-null assertion operator.

```suggestion
          <span v-if="!item.country_code"></span>
          <img v-else-if="isWindows" :alt="codeToFlag(item.country_code).char" :src="codeToFlag(item.country_code).url" :title="item.country" style="max-width: 32px" />
          <span v-else :title="item.country">{{ codeToFlag(item.country_code).char }}</span>
```